pandas/tools/tests/test_util.py

```diff
@@ -401,6 +401,32 @@ def test_downcast(self):
             res = pd.to_numeric(data, downcast=downcast)
             tm.assert_numpy_array_equal(res, expected)
 
+        # check that the smallest and largest values in each integer type pass to each type.
+        int8 = [-128, 127]
```

Comment:
I was actually thinking dictionary if that's okay. And I believe that the numpy types are standard across platforms but I'm not positive which is why I didn't bother with the np.info(dtype).min/max but for safeties sake I'll include it.